spec/amp-actions-and-events.md

```diff
@@ -89,7 +89,7 @@ For example, the following is possible in AMP.
     <th>Description</th>
   </tr>
   <tr>
-    <td>tap</td>
+    <td>`tap`</td>
```

Comment:
Did you preview this file?  I ask because you'll notice that the back ticks don't work as code formatting if it's in HTML code.  Instead, please replace all back ticks with `<code></code>`.